README.md

```diff
@@ -8,7 +8,7 @@ The main functions of Workflow is to efficiently parallelise operations over a s
 
 For examples and more information see [documentation](https://libatoms.github.io/workflow/)
 
-`wfl` its dependendensies may be installed vi `pip install wfl`. 
+`wfl` and its dependendensies may be installed via `pip install wfl`. 
```

Comment:
I may change this :) I'm on a campaign to end the `pip` command (in favor of `python3 -m pip ...`.

Reply:
> What's wrong with pip (if one makes sure it's the correct executable..)?

That's exactly what's wrong.  Especially in systems where there's a `python`(2) and `python3`, it can be `pip3`, etc. Admittedly even assuming `python3` is the right executable is not entirely safe, but I think it's safer.